README.md

```diff
@@ -4,3 +4,12 @@ This Alpine based docker image contains some script to automate stuff after inst
 
 At the moment there are scripts to:
 - Create Personal Access Token
+
+## Use
+
+    docker run -it --rm \
+    -e GITLAB_URL=http://gitlab.company.tld \
+    -e GITLAB_ADMIN_USER=root \
+    -e GITLAB_ADMIN_PASSWD=toor \
+    egeneralov/docker-gitlab-api
```

Comment:
Hey @egeneralov , Maybe just use a generic name here like `docker_image` instead of `egeneralov/docker-gitlab-api`?

Reply:
fixed